v2/jrl-cmakemodules-v2/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+cmake_minimum_required(VERSION 3.22...4.2)
+project(jrl-cmakemodules-v2 VERSION 2.0.0)
+
+include(${CMAKE_CURRENT_SOURCE_DIR}/modules/jrl.cmake)
+
+if(CMAKE_UTILS_BUILD_TESTS)
+    enable_testing()
+    add_subdirectory(tests)
+endif()
+
+if(NOT PROJECT_IS_TOP_LEVEL)
+    return()
+endif()
+
```

Comment:
I don't get it. We will return if included from the main CMakeLists.txt. Then we will not generate the configuration file.